Joseph lunt

GoJoeyGo opened the pull request.

GoJoeyGo committed:
Add files via upload

Updated variables in main book ui and another

GoJoeyGo committed:
Varrible and method names updated to code standard

	isAvailble()       =setAvailble()
	onLoan()	        =setLoan()
	isDamaged()	=setDamaged()
	isBorrowed()	=setBorrowed()
			        =setReturned()
	Repair()	        =setRepair()

GoJoeyGo committed:
Add files via upload

GoJoeyGo committed:
Main

I Updated the variables and methods to the code standards i also modified the code so that it would follow the correct standards ie:
	private static void FIX_BOOKS() {
		new FixBookUI(new FixBookControl()).RuN();		
	}
became 
	private static void fixBook() {
		FixBookControl fixBookControl = new FixBookControl();
		FixBookUI fixBookUI=new FixBookUI(fixBookControl);
		fixBookUI.RuN();	
	}

GoJoeyGo committed:
Add files via upload

I Updated the variables and methods to the code standards i also modified the code so that it would follow the correct standards ie:
	private static void FIX_BOOKS() {
		new FixBookUI(new FixBookControl()).RuN();		
	}
became 
	private static void fixBook() {
		FixBookControl fixBookControl = new FixBookControl();
		FixBookUI fixBookUI=new FixBookUI(fixBookControl);
		fixBookUI.RuN();	
	}

GoJoeyGo committed:
Update Main.java

GoJoeyGo committed:
This is a test to so check somthing

GoJoeyGo committed:
Add files via upload

GoJoeyGo committed:
Merge pull request #2 from mpalme24/GoJoeyGo-patch-1

Add files via upload

GoJoeyGo committed:
Update PayFineUI.java

GoJoeyGo committed:
Update PayFineUI.java

GoJoeyGo committed:
Update PayFineUI.java

GoJoeyGo committed:
Update ReturnBookControl.java

GoJoeyGo committed:
Update book.java

GoJoeyGo committed:
Update book.java

GoJoeyGo committed:
I have used the master and Believe i have updated all files to the correct code Standard

GoJoeyGo committed:
Delete book.java

GoJoeyGo committed:
Delete library.java

GoJoeyGo committed:
Delete member.java

GoJoeyGo committed:
Delete loan.java

GoJoeyGo committed:
Changed title and class from Book.java back to book.java

GoJoeyGo committed:
Update and rename Member.java to member.java

GoJoeyGo committed:
change name back to original

GoJoeyGo committed:
Changed Library back to library

GoJoeyGo committed:
Names and classes updated

GoJoeyGo committed:
Delete Book.java

GoJoeyGo committed:
Delete Library.java

GoJoeyGo committed:
Delete Loan.java

GoJoeyGo committed:
Delete Member.java

GoJoeyGo committed:
Fix conflic

GoJoeyGo committed:
Update book.java

GoJoeyGo committed:
Update book.java

GoJoeyGo committed:
Update book.java

GoJoeyGo committed:
Merge branch 'master' into Joseph-Lunt

MinaTamang reviewed (commented):
c76a17538f3b6a4a0aa94a2505db615b40b6b6cc
Joey can you change boolean Damaged into boolean isDamaged. I think that will look appropriate.

MinaTamang reviewed (changes requested):
@GoJoeyGo  can you change line 32 of Loan.java. getOverDueLoan into boolean isLoanOverDue. I think that will be more appropriate.
Thanks.

MinaTamang commented:
@GoJoeyGo ignore the changes requested message

MinaTamang reviewed (commented):
@GoJoeyGo  can you change line 32 of Loan.java. getOverDueLoan into boolean isLoanOverDue. I think that will be more appropriate.
Thanks.

MinaTamang reviewed (commented):
@GoJoeyGo can you update the line 16 and 17 private List<book> into private List<loan> into private List<Book> and private List<Loan>. I think that's little right way.
And line 53 scanneBook into scanBook or something its just spelling error. Thankyou

MinaTamang reviewed (commented):
@GoJoeyGo can you please change line 9 private scanner keybordinput into keyboardInput aand line 112 public void display into Display().
Thanks

MinaTamang commented on ReturnBookUI.java:
```diff
@@ -3,40 +3,41 @@
-	private Scanner input;
-	private UI_STATE StATe;
+	private ReturnBookControl returnBookUiControl;
+	private Scanner keybordInput;
```
can you change keybordInput into keyboardInput

GoJoeyGo replied:
yep

MinaTamang replied:
cool

MinaTamang commented on BorrowBookControl.java:
```diff
@@ -2,116 +2,111 @@
+
+	private BookControlState borrowBookState;
+
+	private List<book> pendingList;
```
@GoJoeyGo can you change private List<book> and private list<loan> into private List<Book> and <Loan>

GoJoeyGo replied:
what do you mean

MinaTamang replied:
@GoJoeyGo <book> into <Book> and <loan> into <Loan>

MinaTamang replied:
<Book> into <book>

MinaTamang replied:
book into <Book> please

MinaTamang commented on BorrowBookControl.java:
```diff
@@ -2,116 +2,111 @@
+		}
+	}
+
+	public void scanneBook(int bookId) {
```
@GoJoeyGo scanneBook into scanBook just a simple spelling mistake.
Thanks

GoJoeyGo replied:
happy to change thank you for noticing

MinaTamang replied:
no worries @GoJoeyGo

MinaTamang commented on BorrowBookUI.java:
```diff
@@ -1,117 +1,98 @@
-	private Scanner input;
-	private UI_STATE StaTe;
+	private BorrowBookControl control;
+	private Scanner keybordinput;
```
@GoJoeyGo keybordinput into keyboardInput. Please

GoJoeyGo replied:
happy to change thank you for noticing

MinaTamang commented on BorrowBookUI.java:
```diff
@@ -1,117 +1,98 @@
 	}
 
-
 	public void Display(Object object) {
```
@GoJoeyGo Display into display please

GoJoeyGo replied:
display

MinaTamang replied:
yes

MinaTamang commented on FixBookControl.java:
```diff
@@ -1,72 +1,62 @@
-		}	
-		if (MUST_fix) {
-			LIB.Repair_BOOK(Cur_Book);
+	public void setFixBook(boolean bookFixState) {
```
@GoJoeyGo boolean bookFixState into isBookFixed. but I am still not sure can you check in this one as boolean is used to know the true or false so.
Thanks

MinaTamang commented on FixBookUI.java:
```diff
@@ -1,88 +1,70 @@
-	private Scanner input;
-	private UI_STATE StAtE;
+	private FixBookControl fixBookControl;
+	private Scanner keybordInput;
```
@GoJoeyGo keybordInput into keyboardInput. Thankyou

GoJoeyGo replied:
will change keybordInput to keyboardInput i think i did it for all of the code

MinaTamang replied:
heheh yes but not a big deal

MinaTamang commented on Main.java:
```diff
@@ -1,222 +1,211 @@
-		  
-		return sb.toString();
+
+	private static Scanner keybordInput;
```
@GoJoeyGo keybordInput into keyboardInput please.

GoJoeyGo replied:
i just can spell keyboard will change

MinaTamang commented on PayFineControl.java:
```diff
@@ -1,65 +1,59 @@
-	public void Set_UI(PayFineUI ui) {
-		if (!StAtE.equals(CONTROL_STATE.INITIALISED)) {
-			throw new RuntimeException("PayFineControl: cannot call setUI except in INITIALISED state");
+	public void payFineUi(PayFineUI ui) {
```
@GoJoeyGo  payFineUi into setPayFineUi if thats seems okay. Thankyou

GoJoeyGo replied:
payFineUi can be changed to setPayFineUi

MinaTamang replied:
great

MinaTamang commented on ReturnBookControl.java:
```diff
@@ -1,76 +1,76 @@
-	private enum CONTROL_STATE { INITIALISED, READY, INSPECTING };
-	private CONTROL_STATE sTaTe;
+	private ReturnBookUI returnBookUi;
+	private enum RetunrBookControlState { INITIALISED, READY, INSPECTING };
```
RetunrBookControlState into ReturnBookControlState just a spelling mistake @GoJoeyGo thankyou

GoJoeyGo replied:
happy to change

MinaTamang replied:
thanks

MinaTamang commented on ReturnBookControl.java:
```diff
@@ -1,76 +1,76 @@
 	
-	public void Set_UI(ReturnBookUI ui) {
-		if (!sTaTe.equals(CONTROL_STATE.INITIALISED)) {
+	public void returnBookSetUi(ReturnBookUI ui) {
```
@GoJoeyGo I think setReturnBookUi seems little good then returnBookSetUi. can you fix it. Thanks

GoJoeyGo replied:
ReturnBookUi will be changed to setReturnBookUi

MinaTamang replied:
Great

MinaTamang reviewed (commented):
@GoJoeyGo can you please change line 9 private scanner keybordinput into keyboardInput aand line 112 public void display into Display().
Thanks

GoJoeyGo reviewed (commented):
i approve all of the recommend changes and sugestions you have given me mina gits not letting me approve them so i am writing a comment to say that i have reviewed and looked though them all

MinaTamang commented:
> i approve all of the recommend changes and sugestions you have given me mina gits not letting me approve them so i am writing a comment to say that i have reviewed and looked though them all

Okay Joey thanks

GoJoeyGo committed:
Update BorrowBookControl.java

@MinaTamang i cant change the book in Private List "<" book ">" to Book as that changes the file name and causes merging conflicts
@MinaTamang  I changed scannBook into scannBook

GoJoeyGo committed:
Update BorrowBookControl.java

mpalme24 reviewed (changes requested):
Line #86 of BorrowBookControl -  you have borrowBookUi.setBorrowBookUiState -but are then using BorrowBookUI in the brackets. Is this attended? 

Line # 10 of BorrowBookUI - you have "keybordinput" - can this be changed to "keyboardInput"? this is also on line #14 and #21 

Line #66 of BorrowBookUI has "control.scanneBook" - can this be changed to "control.scannedBook"?

Calender.java = fine 

Line #2 of FixBookControl - can fixBookUi be changed to fixBookUI to follow code standards 

This will also need to be changed on line 21, 22, 32, 36, 39

MinaTamang commented:
> > On line #15 and #11 - you have "BookState", could it be more appropriate for it to be bookState?
> 
> Additionally we have
> private String bookTitle;
> private String bookAuthor;
> private String bookCallNumber;
> 
> So changing BookState to bookState would follow the rest of the codes guidelines.

@mpalme24 we can't change BookState to bookState because it is enum. Thankyou

MinaTamang commented:
> Line #86 of BorrowBookControl - you have borrowBookUi.setBorrowBookUiState -but are then using BorrowBookUI in the brackets. Is this attended?
> 
> Line # 10 of BorrowBookUI - you have "keybordinput" - can this be changed to "keyboardInput"? this is also on line #14 and #21
> 
> Line #66 of BorrowBookUI has "control.scanneBook" - can this be changed to "control.scannedBook"?
> 
> Calender.java = fine
> 
> Line #2 of FixBookControl - can fixBookUi be changed to fixBookUI to follow code standards
> 
> This will also need to be changed on line 21, 22, 32, 36, 39

@mpalme24 Its done but I didn't change FixBookUi into FixBookUI sorry. Thanks
